.github/workflows/model_loading.yml

```diff
@@ -0,0 +1,53 @@
+name: Model Loading
+
+on:
+  pull_request:
+    paths:
+      - 'mteb/models/**.py'
+
+jobs:
+  extract-and-run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install mteb[pylate] gritlm mteb[model2vec] xformers
+
+    - name: Extract model names from changed files
```

Comment:
I would actually move this to a bash file in tests (potentially put it all in a folder within tests), and then add a command to the makefile something like `make test-model-loading`.

That way one can reproduce the command without sending it to github.

Reply:
Related comment: https://github.com/embeddings-benchmark/mteb/pull/1698#discussion_r1905174199